File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-05/plagiarized/L1/05/method.kt
package IR

import java.util.Scanner

fun main() {
    print("Enter an integer: ")
    val input = Scanner(System.`in`)
    val number = input.nextInt()
    reverse(number)
}

fun reverse(number: Int) {
    var num = number
    while (num != 0) {
        val remainder = num % 10
        print(remainder)
        num /= 10
    }
    println()
}

File: datasets/IR-Plag/convertai-original-java-rest-python/case-05/plagiarized/L1/05/method.py
import sys

def main():
    print("Enter an integer: ", end="")
    number = int(input())
    reverse(number)

def reverse(number):
    while number != 0:
        remainder = number % 10
        print(remainder, end="")
        number //= 10
    print()

if __name__ == "__main__":
    main()

